Packs/NetscoutAED/Integrations/NetscoutAED/NetscoutAED.yml

```diff
@@ -0,0 +1,1286 @@
+      type: Date
+  - arguments:
+    - default: false
+      description: A single IPv4 host address or CIDR or a comma-separated list of
+        IPv4 host addresses or CIDRs to add.
+      isArray: false
+      name: host_address
+      required: true
+      secret: false
+    - default: false
+      description: A single description that applies to all of the specified hosts
+        or a comma-separated list of descriptions, each of which applies to a specific
+        host.
+      isArray: false
+      name: annotation
+      required: false
+      secret: false
+    deprecated: false
+    description: Adds one or more hosts to the outbound whitelisted list
+    execution: false
+    name: na-aed-outbound-whitelisted-hosts-add
+    outputs:
+    - contextPath: NetscoutAED.OutboundWhitelistHost.annotation
+      description: A description of the host.
+      type: String
+    - contextPath: NetscoutAED.OutboundWhitelistHost.host_address
+      description: IPv4 host address or CIDRs.
+      type: String
+    - contextPath: NetscoutAED.OutboundWhitelistHost.update_time
+      description: The time the host was last updated/set
+      type: Date
+  - arguments:
+    - default: false
+      description: A single IPv4 host address or CIDR or a comma-separated list of
+        IPv4 host addresses or CIDRs to update.
+      isArray: false
+      name: host_address
+      required: true
+      secret: false
+    - default: false
+      description: A single description that applies to all of the specified hosts
+        or a comma-separated list of descriptions, each of which applies to a specific
+        host.
+      isArray: false
+      name: annotation
+      required: false
+      secret: false
+    deprecated: false
+    description: Replaces all the hosts on the outbound whitelisted list.
+    execution: false
+    name: na-aed-outbound-whitelisted-hosts-replace
+    outputs:
+    - contextPath: NetscoutAED.OutboundWhitelistHost.annotation
+      description: A description of the host.
+      type: String
+    - contextPath: NetscoutAED.OutboundWhitelistHost.host_address
+      description: IPv4 host address or CIDRs.
+      type: String
+    - contextPath: NetscoutAED.OutboundWhitelistHost.update_time
+      description: The time the host was last updated/set
+      type: Date
+  - arguments:
+    - default: false
+      description: List of protection group IDs.
+      isArray: false
+      name: pgid
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: 'Set the protection group mode to active (true) or inactive (false).
+        Default: true.'
+      isArray: false
+      name: active
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: 'The protection level (None = use the global protection level,
+        low, medium, high). Default: low'
```

Comment:
Same here, no default. please make sure to add a default value to the arguments if mentioned a default, in all commands

Reply:
Also, something looks off to me in the description - shouldn't we add something like - "options are" ?
@ShirleyDenkberg thoughts?

Reply:
Added default value, waiting for @ShirleyDenkberg response.